po/zh_TW.po

```diff
@@ -18,336 +18,341 @@ msgstr ""
-msgstr ""
+msgstr "更新執行環境到 Gnome 43"
 
 #: data/metainfo.appdata.xml.in:88
 msgid ""
 "Hotfix: fix a crash on startup from trying to access nonexistent settings"
-msgstr ""
+msgstr "修復了一些崩潰視窗"
 
 #: data/metainfo.appdata.xml.in:93
 msgid "Hello, GNOME"
-msgstr ""
+msgstr "Gnome，你好！"
 
 #: data/metainfo.appdata.xml.in:95
 msgid "Whole new UI"
-msgstr ""
+msgstr "全新的界面"
 
 #: data/metainfo.appdata.xml.in:96
 msgid "elementary → GNOME styling"
-msgstr ""
+msgstr "Elementary → Gnome 風格"  
 
 #: data/metainfo.appdata.xml.in:97
 msgid ""
 "Technical upgrade to the latest GNOME platform including GTK4 and LibAdwaita"
-msgstr ""
+msgstr "更新到最新的 Gnome 平台，包括 GTK4 和 libadwata"
 
 #: data/metainfo.appdata.xml.in:103
 msgid "Hello, Odin"
-msgstr ""
+msgstr "奧丁好！"
 
 #: data/metainfo.appdata.xml.in:105
 msgid "Updated for OS 6 and Flatpak"
-msgstr ""
+msgstr "對 OS 6 和 Flatpak 的更新"
 
 #: data/metainfo.appdata.xml.in:106
 msgid "Turkish translations thanks to Safak GENISOL"
-msgstr ""
+msgstr "感謝 Safak GENISOL 翻譯土耳其文"
 
 #: data/metainfo.appdata.xml.in:107
 msgid "Portuguese translations thanks to André Barata"
-msgstr ""
+msgstr "感謝 André Barata 翻譯葡萄牙文"
 
 #: data/metainfo.appdata.xml.in:108
 msgid "Spanish translations thanks to @kevincos5"
-msgstr ""
+msgstr "感謝 @kevincos5 翻譯西班牙文"
 
 #: data/metainfo.appdata.xml.in:114
 msgid "Refreshed icons thanks to Micah Ilbery"
-msgstr ""
+msgstr "感謝 Micah Ilbery 的新圖示"
 
 #: data/metainfo.appdata.xml.in:119
 msgid "Polish translations thanks to Michał Nowakowski"
-msgstr ""
+msgstr "感謝 Michal Nowakoski 翻譯波蘭文"
 
 #: data/metainfo.appdata.xml.in:124
 msgid "Dutch translations thanks to Heimen Stoffels"
-msgstr ""
+msgstr "感謝 Heimen Stoffels 翻譯荷蘭文"
 
 #: data/metainfo.appdata.xml.in:129
 msgid "French translations thanks to @NathanBnm"
-msgstr ""
+msgstr "感謝 @NathanBnm 翻譯法文"
 
 #: data/metainfo.appdata.xml.in:134
 msgid "Clairvoyant is now translatable."
-msgstr ""
+msgstr "現在 ClairVoyant 可以被翻譯了"
 
 #: data/metainfo.appdata.xml.in:139
 msgid "Happy new year! This release contains AppData fixes."
-msgstr ""
+msgstr "新年快樂！此更新包括 AppData 補丁"
 
 #: data/metainfo.appdata.xml.in:144
 msgid "Fix 48px icon"
-msgstr ""
+msgstr "修復 48px 圖示"
 
 #: data/metainfo.appdata.xml.in:149
 msgid "Post-release update"
-msgstr ""
+msgstr "例行更新"
 
 #: data/metainfo.appdata.xml.in:151
 msgid "Clean up AppStream data"
-msgstr ""
+msgstr "清理 AppStream 資料"
 
 #: data/metainfo.appdata.xml.in:152
 msgid "Expand on the description"
-msgstr ""
+msgstr "擴增敘述"
 
 #: data/metainfo.appdata.xml.in:153
 msgid "Change price to $1"
-msgstr ""
+msgstr "更換價格到 30 元"
 
 #: data/metainfo.appdata.xml.in:159 src/FortuneLabel.vala:20
 msgid "It is decidedly so."
-msgstr ""
+msgstr "當然"
 
 #: data/metainfo.appdata.xml.in:161
 msgid "Initial release"
-msgstr ""
+msgstr "最初版本"
 
 #: data/metainfo.appdata.xml.in:167
 msgid "Cassidy James Blaede"
-msgstr ""
+msgstr "Cassidy James Blaede"
 
 #: src/FortuneLabel.vala:16
 msgid "It is certain."
-msgstr ""
+msgstr "很有可能"
```

Comment:
From a quick Google Translate, this says it's "Probably", but this should be a "Yes" answer.

Perhaps I should add translator comments to make these more clear. 😄

Reply:
Thanks! Check out oldtwitter's translation! They have very clear comments.

Reply:
https://github.com/dimdenGD/OldTwitter/pull/488